### DISCUSSION TITLE
in laravel error

### DISCUSSION THREAD

**Discussion by Rohiniprasad88** [spatie/laravel-activitylog#769] [Q&A] [answered]
SQLSTATE[42000]: Syntax error or access violation: 1103 Incorrect table name 'roll' (SQL: create table `students` (`id` bigint unsigned not null auto_increment primary key, `name` varchar(255) not null, `roll`.`no` varchar(255) not null, `standard_id` bigint unsigned not null, `gender` varchar(255) not null, `created_at` timestamp null, `updated_at` timestamp null) default character set utf8mb4 collate 'utf8mb4_unicode_ci')

**Comment by Gummibeer** [comment, marked as answer]
Hey, this error belongs in no way to this package. Possibly you should migrate your database or a migration is missing. But this package has no `roll` and also no `students` table.
